### PR TITLE
Anurag dsa worksheet

### PR DESCRIPTION
**Pull request by yanurag1414**
Adding a solution to the question.
#24

### CODE REVIEW

**Review by lilmistake — APPROVED** [review]

**Comment by lilmistake** [comment]
Your solution to sort012 with Java is correct but the other 2 files are of problems that are either already merged or have issues that need to be fixed (as mentioned in 2 other PRs made by you).
Remove the other 2 files from this PR and I will merge the correct solution. 

Also creating multiple PRs for the same problem is considered spam which is against the guidelines.

File: DSA_Codesheet/Java/delete_nth_from_last.java
Comment (lilmistake) [inline DSA_Codesheet/Java/delete_nth_from_last.java:1]: Problem already solved.

File: DSA_Codesheet/Java/Cycle_Sort.java
Comment (lilmistake) [inline DSA_Codesheet/Java/Cycle_Sort.java:1]: Duplicate PR

**Review by yanurag1414 — COMMENTED** [review]
Removed the other 2 files

**Pull request merged**